Andy Lutomirski reports in 
http://tech.groups.yahoo.com/group/lp_solve/message/9084
A failure to compile in MSVC 8. One problem had to do with regenerating 
lp_rlp.c using a newer Bison. However the following problem remains.



Index: lp_rlp.h
===================================================================
--- lp_rlp.h	(revision 2708)
+++ lp_rlp.h	(working copy)
@@ -22,7 +22,9 @@
 #ifdef __cplusplus
 
 #include <stdlib.h>
+#ifndef _MSC_VER
 #include <unistd.h>
+#endif
 
 /* Use prototypes in function declarations. */
 #define YY_USE_PROTOS
